examples/service-catalog/OWNERS

```diff
@@ -0,0 +1,9 @@
+reviewers:
+  - bparees
+  - pmorie
+  - jboyd01
```

Comment:
@sallyom I guess the `jboyd01/jpeeler` left.

Reply:
i copied this OWNERS from origin/examples. i'll check on it